examples/client/client

```diff
@@ -29,19 +30,30 @@ def build_metadata_dir(base_url: str) -> str:
 
 def init_tofu(base_url: str) -> bool:
     """Initialize local trusted metadata (Trust-On-First-Use) and create a
-    directory for downloads"""
+    directory for downloads
+
+    NOTE: This is unsafe and for demonstration only: the bootstrap root
```

Comment:
Should we log a warning to the user here?

Reply:
I don't think that would be helpful:
* warnings are useful if they allow the user to react somehow. This is not the case here
* the note is there to prevent someone from copy-pasting example code to a real client that should be shipping an embedded root instead